topi/include/topi/transform.h

```diff
@@ -362,6 +362,97 @@ inline Array<Tensor> split(const Tensor& x,
   return result;
 }
 
+/*!
+* \brief strided_slice of a tensor
+*
+* \param x The input tensor
+* \param begin The indices to begin with in the slicing
+* \param end Indicies indicating end of the slice
+* \param stride Specifies the stride values, it can be negative
+* in that case, the input tensor will be reversed in that particular axis
+* \param name The name of the operation
+* \param tag The tag to mark the operation
+*
+* \return A Tensor whose op member is the split operation
+*/
+inline Tensor strided_slice(const Tensor& x,
+                            Array<Expr> begin,
```

Comment:
prefer const reference instead of pass by value when there is no internal move

Reply:
pass by value should only be used in cases where the value is consumed(e.g. by a constructor or move)